BUILDING.md

```diff
@@ -62,7 +62,7 @@ note1 - The gcc4.8-libs package needs to be installed, because node
 
 *Note*: On Windows, running Node.js in windows terminal emulators like `mintty`
   requires the usage of [winpty](https://github.com/rprichard/winpty) for
-  Node's tty channels to work correctly (e.g. `winpty node.exe script.js`).
+  Node's tty channels to work correctly (for example: `winpty node.exe script.js`).
```

Comment:
Nit: while in here, maybe `Node's` -> `Node.js's` or even just remove `Node's` altogether.